modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java

```diff
@@ -740,6 +878,18 @@ public long getNextAssetClassNameId(long groupId) {
 		return classNameId;
 	}
 
+	public long getObjectDefinitionClassNameId() {
+		return getClassNameId(ObjectDefinition.class);
+	}
+
+	public long getObjectFieldClassNameId() {
+		return getClassNameId(ObjectField.class);
+	}
+
+	public long getObjectFolderClassNameId() {
```

Comment:
Is this method being used?

Reply:
Some entities have it (and it's not used explicitly). But I removed it and it works.